pico/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
There is already a LICENSE file at the root of the repo. Is this intended ?

Reply:
There was a point in time where pico was going to be a top-level module. Will discuss it with @tomas-langer - currently it is a child of helidon (obviously), but now that we are close to having our our org on gitHub this might move.

Reply:
Since you are adding pico as a module as part of this PR, the license file should be removed.
You can add the license file back If/when pico moves into its own repository.